lib/main_common.pm

```diff
@@ -103,6 +103,7 @@ our @EXPORT = qw(
   load_security_tests_crypt
   load_security_tests_apparmor_status
   load_security_tests_apparmor_genprof
+  load_security_tests_apparmor_misc
```

Comment:
So now we are going to have 3 apparmor sub-groups, and each sub-group contains a few tests or only 1 test. I have got the total number of apparmor tests that can be automatic from @wnereiz, which are about 10 tests. I am thinking that can we simplify and merge all apparmor tests into 1 group of `load_security_tests_apparmor`? WDYT @wnereiz @okurz

Reply:
There will have **9** cases for AppArmor totally in plan, so I agree with the idea to merge them into one call function. It could reduce complexity and make it easy to manage. 

Even if in future we have more cases (eg. we have 8+ apparmor profile cases in Testopia which have no plan for automation this time), we could still add new call function.

If no other objections, I will establish a new PR for this merge later. Because we should make _load_security_tests_apparmor_misc_ run in **Development** job group to verify everything is okay. So please merge this PR first. Thanks.

Reply:
Yes, this sounds like a good plan